### PR TITLE
Adds line break filter.

### PR DESCRIPTION
**Pull request by jtf621**
@victor-rocha please take a look and see if this is a good approach.
-  check security on untrusted html input from survey form

### CODE REVIEW

**Comment by jtf621** [comment]
#292 proposed fix

File: frontend/app/templates/details.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <div class="col-md-6">
             <div class="school-details">
                 <h2>Mission</h2>
-                <div class="">{{ school.profile.mission_statement }}</div>
```
Comment (victor-rocha) [inline frontend/app/templates/details.html]: you should be able to do this instead: <div class="">{{ school.profile.mission_statement | newline}}</div>
Doesn't really matter but I find this approach more readable.

File: frontend/app/templates/details.html [hunk 2]
```diff
@@ -67,9 +67,7 @@
 
                 <div class="bio">
                     <p>{{ school.profile.principal_name }} (since {{ school.profile.principal_start_year }})</p>
-                    <div class="">
-                        {{ school.profile.principal_bio }}
-                    </div>
+		    <div ng-bind-html="school.profile.principal_bio | newline" class=""></div>
```
Comment (victor-rocha) [inline frontend/app/templates/details.html]: same as above

**Comment by victor-rocha** [comment]
+1 minus minor nitpicks

**Comment by lemanal** [comment]
This almost ready to merge, @jtf621 ?

**Comment by jtf621** [comment]
I found that `ng-bind-html` was required to get the text to render as html.

**Comment by jtf621** [comment]
It is ready @lemanal

**Pull request merged**